examples/strings/strings.bal

```diff
@@ -11,78 +11,77 @@ public function main() {
 
-    //This returns the first index of the first occurrence of the substring within the specified string.
+    // Retrieve the first index of the first occurrence of the substring "on" within the `statement` string.
     int index = statement.indexOf("on");
     io:println("IndexOf: " + index);
 
-    //This returns the first index of the last occurrence of the substring within the specified string.
+    // Retrieve the first index of the last occurrence of the substring "on" within the `statement` string.
     int lastIndex = statement.lastIndexOf("on");
     io:println("LastIndexOf: " + lastIndex);
 
-    //This converts a value of the float type to a string.
-    float value = 5.8;
-    string s4 = <string>value;
-    io:println("ValueOf: " + s4);
-
-    //This replaces the first instance of the 'replacePattern' string with the 'replaceWith' string.
+    // Replace the first instance of the string "Lion" in the `statement` string, with the string "Tiger".
     string s5 = statement.replaceFirst("Lion", "Tiger");
     io:println("ReplaceFirst: " + s5);
 
-    //This replaces the 'replacePattern' string with the replacement string.
+    // Replace the string "Lion" in the `statement` string, with the string "Tiger".
     string s6 = statement.replace("Lion", "Tiger");
     io:println("Replace: " + s6);
 
-    //This replaces each substring of the 'replacePattern' string that matches the given regular expression with the replacement string.
+    // Replace each substring of the `statement` string that matches the given
+    // regular expression with the replacement string "0".
     string s7 = statement.replaceAll("[o]+", "0");
     io:println("ReplaceAll: " + s7);
 
-    //This returns the length of the string.
+    // Retrieve the length of the string.
     int length = statement.length();
     io:println("Length: " + length);
 
-    //This removes any leading and trailing white spaces.
+    // Remove any leading and trailing white spaces.
     string s8 = statement.trim();
     io:println("Trim: " + s8);
 
-    //This checks whether the given string ends with the specified suffix.
+    // Check whether the given string ends with the suffix "Lion".
     boolean hasSuffix = statement.hasSuffix("Lion");
     io:println("HasSuffix: " + hasSuffix);
 
-    //This checks whether the given string starts with the specified prefix.
+    // Check whether the given string starts with the prefix "Lion".
     boolean hasPrefix = statement.hasPrefix("Lion");
     io:println("HasPrefix: " + hasPrefix);
 
-    //This returns an unescaped string by omitting the escape characters of the original string.
+    // Retrieve an unescaped string by omitting the escape characters of the original string.
     string s9 = statement.unescape();
     io:println("Unescape: " + s9);
 
-    //This splits the string with the given regular expression to produce a string array.
+    // Split the string based on the given regular expression to produce a string array.
```

Comment:
```suggestion
    // Split the string using the given regular expression which will produce a string array which contains the results.
```

Reply:
IMO, "to" is better suited here than "which will", since we are asking to split. Also isn't "which contains the results" implied?